csv/read.go

```diff
@@ -23,9 +23,6 @@ func Read(f io.Reader, table *schema.Table, sourceName string, res chan<- []any)
 			}
 			return err
 		}
-		if record[sourceNameIndex] != sourceName {
```

Comment:
why this is removed? also, shouldn't the no header propogate here?

Reply:
This was removed because when there are headers the row does not get read...